src/util/loanSearch/queryParamUtils.js

```diff
@@ -10,7 +10,6 @@ import filterConfig from '@/util/loanSearch/filterConfig';
 export function hasExcludedQueryParams(query) {
 	// Handle temporary query param exclusions
 	const excludedParams = [
-		'activity',
```

Comment:
I don't think we can change this exclusion, since it will affect the search bar navigation for users not in the experiment. For the experiment, we should probably replicate this method for the filter page.

Reply:
Nice call...

Reply:
On this particular case, we are not showing Activities in search bar results, even that, with activities support on filter page, we should be able to show them and don't redirect to legacy filters that is the case we are currently using this function. thoughts?

Reply:
That's right, we are not showing results in the search bar with activities. Also, the usage to exclude that query param is to redirect to legacy filter if activity included, a  feature we are enabling now on filter page.

Reply:
Sounds good. I was suggesting to create a new version of this method for the experiment, so it would help with new navigation, but if there's no search with activities, we're good. We just need to make sure to not affect existing functionality, and it sounds like we're good here.